client/src/app/admin/panel/settings/admin-panel-advanced-settings.js

```diff
@@ -121,13 +119,21 @@ class AdminPanelAdvancedSettings extends React.Component {
 
     renderKey() {
         let currentAPIKey = this.state.APIKeys[this.state.selectedAPIKey];
-
         return (
             <div className="admin-panel-advanced-settings__api-keys__container-info">
                 <div className="admin-panel-advanced-settings__api-keys-subtitle">{i18n('NAME_OF_KEY')}</div>
                 <div className="admin-panel-advanced-settings__api-keys-data">{currentAPIKey.name}</div>
                 <div className="admin-panel-advanced-settings__api-keys-subtitle">{i18n('KEY')}</div>
                 <div className="admin-panel-advanced-settings__api-keys-data">{currentAPIKey.token}</div>
+                <div className="admin-panel-advanced-settings__api-keys-subtitle">{i18n('PERMISSIONS')}</div>
+                <div className="admin-panel-advanced-settings__api-keys__permissions">
+                    <FormField className="admin-panel-advanced-settings__api-keys__permissions__item" value={currentAPIKey.canCreateTickets*1} label={i18n('TICKET_CREATION_PERMISSION')} field='checkbox'/>
+                    <FormField value={currentAPIKey.shouldReturnTicketNumber*1} label={i18n('TICKET_NUMBER_RETURN_PERMISSION')} field='checkbox'/>
```

Comment:
Do we really need the `*1`? I checked the codebase and we are using them a lot, but we probably would be benefitted from using them only where needed. Ditto for all others `*1`.